jenkins/docker/Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+pipeline {
```

Comment:
Seems like this job needs to be manually triggered, can we automatically trigger this process on docker changes to merged into the repo?

Reply:
We can add scm as an improvement to monitor specific files or folder. eg. docker.

Reply:
Can we make sure that #801 stays open then, this is much much closer, but not all the way there unless you would prefer to open a new issue.

Reply:
Added.